public/js/src/module/connection.js

```diff
@@ -355,6 +354,17 @@ function getMaximumSubmissionSize( survey ) {
         .then( () => survey );
 }
 
+/**
+ * @param {string} basePath
+ * @param {string} [enketoId]
+ * @return {string}
+ */
+const getTransformURL = ( basePath, enketoId ) => {
```

Comment:
Does this need to become public?

Reply:
It's not being exported. For context (sorry), we've been gradually moving toward inline `export` statements rather than `_` prefixes to signify module-private. Besides being a more idiomatic style, it also significantly improves the ability to navigate references across modules.